extensions-core/s3-extensions/src/test/java/org/apache/druid/storage/s3/S3StorageConnectorProviderTest.java

```diff
@@ -158,7 +159,8 @@ public void configure(Binder binder)
                 new S3UploadManager(
                     new S3OutputConfig("bucket", "prefix", EasyMock.mock(File.class), new HumanReadableBytes("5MiB"), 1),
                     new S3ExportConfig("tempDir", new HumanReadableBytes("5MiB"), 1, null),
-                    new DruidProcessingConfigTest.MockRuntimeInfo(10, 0, 0))
+                    new DruidProcessingConfigTest.MockRuntimeInfo(10, 0, 0),
+                    new StubServiceEmitter("service", "host"))
```

Comment:
You may also use the zero-arg constructor of `StubServiceEmitter` since the passed args are not meaningful anyway. Same comment for the other usages.